example/lib/generated_plugin_registrant.dart

```diff
@@ -5,8 +5,7 @@
 // ignore_for_file: directives_ordering
 // ignore_for_file: lines_longer_than_80_chars
 
-// ignore: implementation_imports
-import 'package:flutter_segment/src/segment_web.dart';
```

Comment:
Why adding a file that exports the same import will fix the warning? is it the real solution for this?

Reply:
@T-moz Ping